### DISCUSSION TITLE
[ios] Support Sharing

### DISCUSSION THREAD

**Discussion by vHanda** [GitJournal/GitJournal#754] [Feature Requests]
Sharing to the GitJournal ios app is only implemented in Android so far.

---
> Please vote on this issue with a 👍 reaction. It's one of the primary mechanisms used to prioritize issues. Please also subscribe to the issue if you're interested, as that way I can discuss possible implementations and share mockups.

**Comment by ScootRay** [comment]
Thanks for posting this, missed your email.

**Reply by oevl** [reply]
Long time Android user, bought it today for iOS, and was confused because sharing wasn't working, had to search the issues and found this thread, hope sharing can be implemented.
